Reorganize `/static/assets/` directory

EthanThatOneKid opened the pull request:
### Changes

- Move all files in `/static/assets/authors/` to `/static/people/`.
- Move all remaining files to `/static/assets/`.
- Replace renamed file names in source code.

Resolves #997.

EthanThatOneKid committed:
remove assets

EthanThatOneKid committed:
add svg assets

EthanThatOneKid committed:
add quiz assets

EthanThatOneKid committed:
add pomo asset

EthanThatOneKid committed:
add png assets

EthanThatOneKid committed:
add jpg assets

EthanThatOneKid committed:
wip

EthanThatOneKid committed:
add cssg assets

EthanThatOneKid committed:
add 1st assets

EthanThatOneKid committed:
add people

TODO: Open new issue in [Boardd](https://oss.acmcsuf.com/boardd) related to changing the location of the image assets for people.

k4rni reviewed (changes requested):
Couple of assets are not showing up:
<img width="258" alt="image" src="https://github.com/EthanThatOneKid/acmcsuf.com/assets/60043611/df42bff7-0023-4ab3-b40d-39b0aeb5f3b8">
![image](https://github.com/EthanThatOneKid/acmcsuf.com/assets/60043611/6e84fbd5-7c19-47b4-ba25-1f079fb01dee)
![image](https://github.com/EthanThatOneKid/acmcsuf.com/assets/60043611/ced606e6-3edb-4ff0-9819-335a4b953485)

EthanThatOneKid committed:
fix assets mistakes

Resolves <https://github.com/EthanThatOneKid/acmcsuf.com/pull/1001#pullrequestreview-1769377496>.

EthanThatOneKid committed:
Merge branch 'main' into fix/997

EthanThatOneKid committed:
fix officers

EthanThatOneKid committed:
Merge branch 'main' into fix/997

k4rni reviewed (approved):
LGTM!
